### PR TITLE
adding MolCLR baseline for Polaris

### PR DESCRIPTION
**Pull request by akshatzalte**
Main addtions are three python scripts:

1. `create_benchmark_csv.py` that uses the Polaris environment to load the benchmarks and then save the csvs for training and test datasets. It also creates and stores the metadata that can be used for downstream applications.
2. `train_model.py` script that finetunes the MolCLR model (pretrained GCN model, best from their paper) on all the polaris benchmarks and store all the predictions for the test set for 5 random seed (train/val split)
3. `evaluate.py` script that is analogous to other `evaluate.py` scripts in this project. Instead of model training, it simply fetches the predictions output by the `train_model.py`

Finally, the `MolCLR.md` file has the results in the same format as other models and can be used to recreate the HSD plots (@JacksonBurns). I would want a quick review before we run the analysis and get the final figures.

TODO:
Add MolCLR MoleculeACE benchmarks.

### CODE REVIEW

**Comment by akshatzalte** [comment]
I checked the code, cleaned it, and ensured consistency in writing the corresponding README. Merging it!
